Runtime/Connection/SerialDevice/SerialDeviceConnection.cs

```diff
@@ -64,6 +64,7 @@ public override Task ConnectAsync()
                 WriteTimeout = 0 == serialDeviceProperties.WriteTimeoutMS ?
                     SerialPort.InfiniteTimeout :
                     serialDeviceProperties.WriteTimeoutMS,
+                ReadTimeout = 3000,

```

Comment:
Should be added as part of the serialDeviceProperties not hardset to keep it configurable